appveyor/test_task.bat

```diff
@@ -90,8 +90,10 @@ mkdir c:\tests_tmp
 
 set TEST_PHP_JUNIT=c:\junit.out.xml
 
+set SKIP_SLOW_TESTS=1
+
 cd "%APPVEYOR_BUILD_FOLDER%"
-nmake test TESTS="%OPCACHE_OPTS% -q --offline --show-diff --show-slow 1000 --set-timeout 120 -g FAIL,XFAIL,BORK,WARN,LEAK,SKIP --temp-source c:\tests_tmp --temp-target c:\tests_tmp"
```

Comment:
Why is the `--show-slow` dropped here?

It might be interesting to do a run with `--show-slow 0` to get a full list of test timings, including the "fast" ones (which are probably comparatively slow on AppVeyor).

Reply:
Oh, I've mistankenly assumed that `--show-slow` is only about slow tests.

Anyhow, it seems skipping the slow tests [saves quite some time](https://ci.appveyor.com/project/php/php-src/builds/22075045).